src/django/api/migrations/0158_create_moderation_events_table.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.17 on 2024-10-14 10:04
+
+import django.contrib.postgres.fields
+import django.contrib.postgres.indexes
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0157_delete_endpoint_switcher_for_list_uploads'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ModerationEvent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='Date when the moderation queue entry was created.')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='Date when the moderation queue entry was last updated.')),
```

Comment:
It looks like we must indexing it indeed because of `ORDER BY
  af.updated_at ASC` in `src/logstash/sql/sync_production_locations.sql`. I've added necessary attribute to the model.